usermods/usermod_v2_four_line_display_ALT/readme.md

```diff
@@ -1,43 +1,31 @@
 # I2C/SPI 4 Line Display Usermod ALT
 
 Thank you to the authors of the original version of these usermods. It would not have been possible without them!
-"usermod_v2_four_line_display"
-"usermod_v2_rotary_encoder_ui"
+"usermod_v2_four_line_display" (old but gold, removed since release 0.15.0)
```

Comment:
There is no need to add personal opinion. As well below.

Reply:
This readme is referring to something that does not exist anymore. Maybe is it better to remove the whole reference?

Reply:
"old but gold" is a personal opinion.